benchmarks/kernels.py

```diff
@@ -0,0 +1,99 @@
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+
+import jax.random as jr
+
+from gpjax import kernels
+
+
+class Kernels:
+    param_names = ["n_data", "dimensionality"]
+    params = [[10, 100, 500, 1000, 2000], [1, 2, 5]]
+
+    def setup(self, n_datapoints, n_dims):
+        key = jr.PRNGKey(123)
+        self.X = jr.uniform(
+            key=key, minval=-3.0, maxval=3.0, shape=(n_datapoints, n_dims)
+        )
+
+
+class RBF(Kernels):
+    def setup(self, n_datapoints, n_dims):
+        super().setup(n_datapoints, n_dims)
+        self.kernel = kernels.RBF(active_dims=list(range(n_dims)))
+
+    def time_covfunc_call(self, n_datapoints, n_dims):
```

Comment:
I suppose it doesn't work if we implement this on the parent class `Kernels` instead of on each subclass?

Reply:
Unfortunately not. That would be a cool feature.